pkg/e2e/operators/operators.go

```diff
@@ -517,7 +517,7 @@ Loop:
 			return nil, err
 		default:
 			if elapsed < timeoutDuration {
-				log.Printf("Waiting %v for %s deployment to exist", (timeoutDuration - elapsed), deploymentName)
+				log.Printf("Waiting %v for %s deployment to exist", timeoutDuration-elapsed, deploymentName)
```

Comment:
Could we change this back to having the space? 
Removing the space between variables makes it hard to read, I thought it was one word.